package.json

```diff
@@ -1,11 +1,8 @@
 {
-  "name": "@prb/foundry-template",
-  "description": "Foundry-based template for developing Solidity smart contracts",
+  "name": "@vacp2p/foundry-template",
+  "description": "Foundry-based template for developing Solidity smart contracts used by Vac",
   "version": "1.0.0",
-  "author": {
-    "name": "Paul Razvan Berg",
-    "url": "https://github.com/PaulRBerg"
-  },
+  "author": {},
```

Comment:
Maybe add yourself as the author here, and link to https://vac.dev?

Reply:
Done